.gitignore

```diff
@@ -0,0 +1,20 @@
+*.annot
+*.cmo
+*.cma
+*.cmi
+*.a
+*.o
+*.cmx
+*.cmxs
+*.cmxa
+
+# ocamlbuild working directory
+_build/
+
+# ocamlbuild targets
+*.byte
+*.native
+
+# oasis generated files
+setup.data
+setup.log
```

Comment:
Should we add `setup.ml` too, as it is not under version control anymore?
Also, I think `myocamlbuild.ml` and `_tags` (and possibly other files) should also be ignored, these are also generated by OASIS:
https://ocaml.org/learn/tutorials/setting_up_with_oasis.html#TheOASISfiles

Also, since we use the `DevFiles` plugin, I think we should ignore `Makefile` and `configure` too.